site/committers.html

```diff
@@ -162,6 +162,9 @@
       <h5>Latest News</h5>
       <ul class="list-unstyled">
         
+          <li><a href="/news/spark-2-4-3-released.html">Spark 2.4.3 released</a>
+          <span class="small">(May 08, 2019)</span></li>
```

Comment:
This also doesn't match with the download page, `05/07`, too. Let's fix this to `(May 06, 2019)`, consistently.

Reply:
This is the news post date. That is today.